richtext/plugins/org.polarsys.kitalpha.richtext.widget/src/org/polarsys/kitalpha/richtext/widget/factory/MDERichTextFactory.java

```diff
@@ -267,10 +268,12 @@ private void removeUselessItemFromToolbar() {
 	protected void addEditorToolbarItems(MDENebulaBasedRichTextWidget widget) {
 		widget.addToolbarItem(widget, MDERichTextConstants.MDE_SAVE, MDERichTextConstants.MDE_SAVE,
 				"Save editor content", MDERichTextConstants.MDE_SAVE_TOOLBAR, Constants.SAVE_IMAGE_ICON, new SaveContentHandler());
-
+		
+		if (Platform.OS_LINUX.equals(Platform.getOS()) && !Boolean.getBoolean("org.polarsys.kitalpha.richtext.webkit.gtk.compatibility.workaround.disable")) {
```

Comment:
The Refresh toolbar action triggers a deadlock on Linux.
And it seems to have no utility.
This is not a fix, this commit like the 2 parent commits are a workaround to avoid identified deadlocks with the RichText editor when Capella 7.x is launched on Linux.

Reply:
fixed in new PR update